deno/rest/v10/mod.ts

```diff
@@ -844,6 +844,182 @@ export const Routes = {
 	},
 };
 
+export const CDNRoutes = {
+	/**
+	 * Route for:
+	 * - GET `/emojis/{emoji.id}.{png|jpeg|webp|gif}`
+	 *
+	 * As this route supports GIFs, the hash will begin with `a_` if it is available in GIF format
+	 */
+	emoji(emojiId: Snowflake, format: 'png' | 'jpeg' | 'webp' | 'gif') {
```

Comment:
Can you export a type for the formats? Or better yet make them an enum

Reply:
I tried this for it limit the formats, but i get this error:
![image](https://user-images.githubusercontent.com/79790819/174615152-3c7b2568-64e6-4639-b89e-6727847a48c7.png)
![image](https://user-images.githubusercontent.com/79790819/174615155-adcd724e-8b24-4e57-b797-604fd9d34b0f.png)

So there is no limitation, but I added a more descripitive JSDoc:
Example: `This route supports the extensions: PNG, JPEG, WebP, GIF`

Reply:
`Exclude<ImageFormat, ImageFormat.Lottie>`